tests/functional_tests/ffront_tests/test_execution.py

```diff
@@ -1164,7 +1164,11 @@ def program_domain(
         lower_j: int64,
         upper_j: int64,
     ):
-        fieldop_domain(a, out=a, domain={IDim: (lower_i, upper_i), JDim: (lower_j, upper_j)})
+        fieldop_domain(
```

Comment:
The specification of the backend is not correct in this file. Since `@program` has no backend specified it will fallback to the embedded one, while it should use `fieldview_backend` (which is only specified on the field operator). We should add a check for this at some point. If you specify the backend this test will fail for gtfn. Please use `pytest.skip` for that case and add another test that doesn't use `floordiv` and `pow` which are not (fully) supported in gtfn.